jobserver/urls.py

```diff
@@ -246,6 +246,11 @@
 
 project_urls = [
     path("", ProjectDetail.as_view(), name="project-detail"),
+    path(
+        "analyses/",
```

Comment:
Having this at the project level feels like the right thing to do, but I don't know how will link from the project page, whereas if we have it at the workspace level, we could easily insert something between Workspace purpose and Releases for Interactive Reports/Analyses. What do you think?

Maybe we need to let Tom O have a think about how the user journey will work before making any changes.

Reply:
It'll probably start life as a card, at the bottom of which ever column seems the least daft.  It's definitely not ideal but I know Tom is planning to re-think this page with timeline-style cards which would be perfect for this.